Dockerfile

```diff
@@ -3,9 +3,11 @@ RUN apk add -U --no-cache ca-certificates && update-ca-certificates
 
 FROM scratch
 EXPOSE 3592 3593
+ENV CERBOS_CONFIG="/conf.default.yaml"
 VOLUME ["/policies"]
 ENTRYPOINT ["/cerbos"]
-CMD ["server", "--config=/conf.default.yaml"]
+CMD ["server"]
+HEALTHCHECK --interval=1m --timeout=3s CMD ["/cerbos", "healthcheck"]
```

Comment:
Ah! Good catch.

Reply:
I just set the default timeout to 2s because it should never take 10s anyway. If users want to override it, they can do that in `docker run` by setting `--health-timeout=<value>` and `-e CERBOS_HC_TIMEOUT=<value>`.